data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	Only when you're with him inside the Arach vessel is that you think of asking if he even knows how to fly a ship, but he reassures you that that won't be a problem.`
+			`	"To work as a copilot for an old friend, my first job was. Worry about me crashing the ship, you need not," he jokes.`
+			`	He says he will wait for your lead, and follow you to <destination> once you depart.`
+				accept
+	npc accompany save
+		personality escort timid
+		government "Coalition"
+		ship "Arach Courier" "Boop"
+	on visit
+		dialog `You have reached <planet>, but Inturi's ship hasn't arrived yet. Better depart and wait for it to get here.`
+	on complete
+		"coalition jobs" ++
+		event "saryd couple follow-up" 51 96
+		conversation
+			`You help Inturi park the ship on the plot of land, but since the Arach Courier takes up almost the entire area, your own ship has to land on a regular landing pad.`
+			`	After taking a transport up the mountain, you see that some workers are already there, and are starting to remove ship components that won't be needed, like its engines and the hyperdrive. A house doesn't need to be able to traverse the void of space, after all.`
+			`	The couple comes meet up with you, "In a day or two, after some shopping we do, fully furnished the house will be," Aunaris says, looking at the ship.`
+			`	"Helped us a lot in getting it all sorted out you have, Captain <last>. Focus on working hard to pay you back in full, we will," Inturi adds.`
+			choice
+				`	"You don't need to worry about paying me, I helped because I wanted to, not for a payday."`
+				`	"Well, I hope everything goes well. Best of luck to you two!"`
+					goto end
+			apply
+				set "refused saryd couple payment"
+			`	They look at you confused and then try to insist on paying you, but after some minutes they agree. "Too kind a heart you are, Captain," Aunaris says.`
+			label end
+			`	They thank you again, and you head back to your ship.`
+
+
+
+event "saryd couple follow-up"
+
+
+
+mission "Saryd Couple Follow-Up A"
+	invisible
+	landing
+	to offer
+		has "event: saryd couple follow-up"
+		not "refused saryd couple payment"
+	source
+		government "Coalition" "Heliarch"
+	on offer
+		payment 325400
+		conversation
+			`When you land, you're notified of <payment> being transferred to your account.`
+			`	With it, there's a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
+			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there, after people saw their home was a spaceship.`
```

Comment:
```suggestion
			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
```
No comma b/c dependent clause following independent clause. `The couple's home` is a personal preference.